ph5/clients/ph5tostationxml.py

```diff
@@ -910,6 +907,10 @@ def main():
                       nsmap={'iris': "http://www.fdsn.org/xml/station/1/iris"})
         elif out_format == "KML":
             inv.write(args.outfile, format='KML')
+        elif out_format == "SACPZ":
+            inv.write(args.outfile, format="SACPZ")
+        elif out_format == "STATIONTXT":

```

Comment:
I would change this to "TEXT" in order to remain consistent with the web services.

Reply:
good idea. I'll fix that.